packages/manager/apps/telecom/src/app/telecom/pack/move/offers/move-offers.controller.js

```diff
@@ -210,10 +225,54 @@ export default class PackMoveOffersCtrl {
     set(offer, 'displayedPrice', displayedPrice);
   }
 
+  static updateSelectedGtrOption(value, offer, optionName) {
+    angular.forEach(offer.options, (option) => {
+      if (
+        option.name.match(/^gtr_/) &&
+        option.selected !== null &&
+        optionName.match(/^gtr_/)
+      ) {
+        if (option.name !== optionName) {
+          option.selected = false; // eslint-disable-line no-param-reassign
```

Comment:
**question:** Would it be possible to not disabling ESLint rule?

Reply:
No I can't, we can only select one gtr option at a time